### DISCUSSION TITLE
rar архивы в версии 2.2 открываются в сторонней программе

### DISCUSSION THREAD

**Discussion by badmansan** [elfmz/far2l#1039] [Q&A] [answered]
Здравствуйте, установил far2l через apt (там версия 2.2), пробую зайти в rar архивы. В мелкие заходит нормально, а в большие - нет, отправляет в стороннюю программу (ark в моём случае). Поставил последний билд с гита, попробовал - с ним всё хорошо, заходит в любые архивы. Это какая-то настройка multiarc, или где-то зашит предел размера? Почему в 2.2 не работает, а в 2.3 - работает?

**Comment by elfmz** [comment, marked as answer]
Потому что в мультиарке было довольно много изменений, в том числе для поддержки новых форматов. Вот например скорее всего то что вас интересует - https://github.com/elfmz/far2l/commit/5773853770308b96da30b7227681bd98403cb252

**Reply by badmansan** [reply]
Пару дней назад обновился репозиторий apt до версии 2.3, сейчас всё хорошо. Спасибо за вашу работу!
